### DISCUSSION TITLE
Currency Calculator Background

### DISCUSSION THREAD

**Discussion by JeongseopYun** [Satish-Mutyala/T24S2T1EXTG12#1] [General]
Hello team, I have created the Currency Calculator but is there any color that you guys want me to put? I use the same colour as the home page. 

![Cal photo](https://github.com/user-attachments/assets/996168cd-b441-4a84-953d-08de2c0df540)

**Comment by Satish-Mutyala** [comment]
@JeongseopYun The HomePage is using the colors as per the screenshot below. Would be great if we can have same colors across all calculators to maintain the consistency. 
![image](https://github.com/user-attachments/assets/fa6842b4-2ae5-4435-bfdd-d0955cf4d397)

**Comment by WeiZhou-dev** [comment]
@JeongseopYun  Hi Jeongseop, it is nice to use the same colour as the home page to keep the consistency.
